### DISCUSSION TITLE
Anyway to disable community posts?

### DISCUSSION THREAD

**Discussion by lyrashmi** [arichornlover/uYouEnhanced#128] [Q&A]
Every time I start the app it is one of the first things I see on the home page and I'd rather just disable them completely.

**Comment by yodaluca23** [comment]
Settings > General > Hide Community Posts

**Reply by Shakti-7093** [reply]
yes trying too

**Reply by arichornlover** [reply]
I’m removing that option. Seems like it is starting to not work correctly, I might bring it back next release hopefully.
